CMakeModules/Doc.cmake

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
This command may cause problems with [policies](https://cmake.org/cmake/help/v3.0/command/cmake_minimum_required.html) so I would avoid using it at other places than the main `CMakeLists.txt`.

Reply:
this is not the only module using it, so I've added a separated commit removing it from all the cmake modules in libyang.

Reply:
Okay, that should probably be our policy from now.